app/models/activity.rb

```diff
@@ -133,6 +133,13 @@ def parent_level
     Activity.levels.keys[existing_level_index - 1]
   end
 
+  def child_level
+    existing_level_index = Activity.levels.keys.index(level)
```

Comment:
The reliance on the ordering of a hash here (`enum :level`) makes me nervous. I know Ruby hashes are insertion order but it's quite easy to break things that rely on this by accident. Is there a more robust way we could encode the hierarchy?

Reply:
@jcoglan I tried a couple of approaches, but in the end settled for a [far simpler solution ](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/564/commits/c79e0a0c2c085a5b85fa1d334cff92749ed538e7).

Reply:
That looks a lot more robust and less easy to break accidentally 👍 

This is also made more robust by the fact we store levels by name, not by implicitly mapping them to integers in the database which would make it more sensitive to the `enum` being re-ordered.